src/components/User/UserComponent.jsx

```diff
@@ -65,48 +67,52 @@ const UserDetailsComponent = (props) => {
         <Table.Footer>
           <Table.Row>
             {!props.emptyUsersList() && (
-              <Table.HeaderCell colSpan="4" id="pagination-header">
-                <Segment.Group horizontal id="art-pagination-section">
-                  <Segment>
-                    <Pagination
-                      id="art-pagination-component"
-                      totalPages={props.handlePageTotal()}
-                      onPageChange={props.handlePaginationChange}
-                      activePage={props.activePage}
-                    />
-                  </Segment>
-                  <Segment>
-                    <DropdownComponent
-                      id="page-limit"
-                      placeHolder="Show Rows"
-                      options={rowOptions}
-                      upward
-                    />
-                  </Segment>
-                </Segment.Group>
-              </Table.HeaderCell>
-            )}
+            <Table.HeaderCell colSpan="4" id="pagination-header">
```

Comment:
The whole change to `<Table.HeaderCell />` is unnecessary and it messes the indentation. Please revert to how it was before and only add the needed props of `value` and `onChange` to `DropdownComponent`